protocol/examples/messageData/election_open.json

```diff
@@ -0,0 +1,7 @@
+{
+    "object": "election",
+    "action": "open",
+    "lao": "fzJSZjKf-2cbXH7kds9H8NORuuFIRLkevJlN7qQemjo=",
+    "election": "zG1olgFZwA0m3mLyUqeOqrG0MbjtfqShkyZ6hlyx1tg=",
```

Comment:
is this based on the same data as the other example jsons ? 😄

Reply:
Not sure tbh as this is copied from #657 😅